modules/RTC/RTCUtils.js

```diff
@@ -872,6 +865,21 @@ class RTCUtils extends Listenable {
                         return this.audioTracks;
                     };
                 }
+
+                this.p2pPcConstraints
+                    = JSON.parse(JSON.stringify(this.pcConstraints));
+
+                // Allows sending of video to be suspended if the bandwidth
+                // estimation is too low.
+                if (!options.disableSuspendVideo) {
```

Comment:
No, the option affects the bridge connection. For p2p we enable it unconditionally.